SofaKernel/modules/SofaTypes/src/sofa/defaulttype/Vec.h

```diff
@@ -0,0 +1,101 @@
+/******************************************************************************
```

Comment:
IMHO, the path should be /src/sofa/types/Vec.h and namespace sofatypes::Vec or sofa::types::Vec.h (but this one is not consistant with the way we will should name plugins namespaces).